src/parser/jobs/ast/modules/EarthlyStar.tsx

```diff
@@ -94,14 +93,14 @@ export default class EarthlyStar extends Module {
 		*/
 		const holdDuration = this.uses === 0 ? this.parser.fightDuration : this.totalHeld
 		const usesMissed = Math.floor(holdDuration / (this.data.actions.EARTHLY_STAR.cooldown * 1000))
-		if (usesMissed > 1 || this.uses === 0) {
+		if (usesMissed > 1) {
```

Comment:
This should prob be `>= 1`, since your severities include 1. It's a tiered suggestion tho, so you don't need the conditional at all. If usesMissed is 0, the suggestion won't get shown since it's below minimum severity.